requests_ratelimiter/requests_ratelimiter.py

```diff
@@ -72,6 +73,7 @@ def __init__(
         self.limit_statuses = limit_statuses
         self.max_delay = max_delay
         self.per_host = per_host
+        self.bucket = bucket
```

Comment:
If I understand your use case correctly, I think `per_host=False` will already do what you want.

Adding the `bucket` option sounds good to me, but that could be combined with `_default_bucket`. And that could be made a public attribute, so:
```py
self.default_bucket = bucket or str(uuid4())
```

Then I think the rest of the changes below won't be needed, if you're okay with setting `per_host`:
```python
sessionA = LimiterSession(per_second=5, per_host=False, bucket='tenant1')
sessionB = LimiterSession(per_second=5, per_host=False, bucket='tenant2')
```